docs/br/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Home
+meta:
+  - name: description
+    content: Ethereum é uma plataforma global e descentralizada de dinheiro e novos tipos de aplicativos. No Ethereum, você pode escrever um código que controla dinheiro e criar aplicativos acessíveis em qualquer lugar do mundo.
+  - property: og:title
+    content: Home | Ethereum
+  - property: og:description
+    content: Ethereum é uma plataforma global e descentralizada de dinheiro e novos tipos de aplicativos. No Ethereum, você pode escrever um código que controla dinheiro e criar aplicativos acessíveis em qualquer lugar do mundo.
+lang: pr-BR
+layout: home
+---
+
+<HomePage/>
```

Comment:
In order to complete this homepage translation, we'll need to add the text & links in the translations.js file. See the existing sections as examples: https://github.com/ethereum/ethereum-org-website/blob/dev/docs/.vuepress/theme/utils/translations.js#L5